UnityProject/Assets/Scripts/Objects/Pipes/Devices/PassiveGate.cs

```diff
@@ -12,7 +12,7 @@ public class PassivePump : MonoPipe
 
 		[NonSerialized] public float MaxPressure = 4500f;
 		[NonSerialized] public float ThresholdPressure = 10f;
-		[NonSerialized] public float TargetPressure = 101.325f;
+		[NonSerialized] public float TargetPressure = AtmosConstants.ONE_ATMOSPHERE;
```

Comment:
you could just use `AtmosConstants.ONE_ATMOSPHERE` wherever you are using `TargetPressure`

Reply:
TargetPressure can be changed via nettab, the default value is just AtmosConstants.ONE_ATMOSPHERE

Reply:
ah that makes sense